src/schema/submission.yaml

```diff
@@ -78,6 +78,37 @@ classes:
         inlined: true
         inlined_as_list: true
 
+  # Parameter:
```

Comment:
I think we should delete the commented out lines if we are sure that they are not needed.

Reply:
agreed - will keep this conversation open until we are sure that we don't need the parameter definition